config.py

```diff
@@ -162,7 +162,7 @@ class DevelopmentConfig(BaseConfig):
     DEBUG = True
 
     #: Logger setup
-    LOG_LEVEL = logging.DEBUG
+    LOG_LEVEL = logging.INFO
```

Comment:
nit: undo this?

Reply:
`DEBUG` also includes a lot of `fsevents` debug logic which clogs the local logs. See #300.